.changes/next-release/Feature-8729574b-6dd2-4ab4-88bb-a5f13b710e0b.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "API Gateway nodes include their ID in the label"
```

what this means for the customer :
```suggestion
	"description": "List API Gateway names with their ID (so Toolkit can list APIs with identical names)"
```